android/sdl_android/src/androidTest/java/com/smartdevicelink/test/VehicleDataHelper.java

```diff
@@ -5,16 +5,19 @@
 import com.smartdevicelink.proxy.rpc.BodyInformation;
```

Comment:
@kostyaBoss I think you can revert these changes, as vehicle data does not need to be changed for this PR. The corresponding PR for the JavaScript suite (https://github.com/smartdevicelink/sdl_javascript_suite/pull/354/files) did not make changes to the vehicle data.